e2e/e2e_tests.py

```diff
@@ -0,0 +1,151 @@
+import os
+from time import sleep
+import unittest
+import subprocess
+
+try:
+    import requests
+    from bs4 import BeautifulSoup
+except ImportError:
+    requirements_file = os.path.join(os.path.dirname(__file__), 'requirements.txt')
+    subprocess.check_call("pip install -r {}".format(requirements_file).split(' '))
+    import requests
+    from bs4 import BeautifulSoup  # noqa ignore imported but unused
+
+from helpers.html_helpers import make_soup
+from helpers.zip_file_helpers import check_if_result_contains_data
+from helpers import docker_compose
+
+ADMIN_USER_FOR_TESTS = 'admin'
+ADMIN_PASSWORD_FOR_TESTS = 'admin'
+
+
+def _start_containers():
+    """
+    first clean up, then check for newer images and then start the containers.
+    Then wait 10 seconds to be certain the webapp is up and ready to receive.
+    """
+    docker_compose.clean()
+    docker_compose.pull()
+    docker_compose.start()
+    docker_compose.create_superuser_for_test(username=ADMIN_USER_FOR_TESTS, password=ADMIN_PASSWORD_FOR_TESTS)
+    sleep(10)
+
+
+def _stop_containers():
+    """
+    stop the containers and then cleanup
+    """
+    # docker_compose.stop()
+    # docker_compose.clean()
```

Comment:
With these lines commented out, this method does nothing. It still gets called from `if __name__ == '__main__':`, so it probably _should_ do _something_, I guess?